lib/Address.php

```diff
@@ -68,9 +68,27 @@ public function getLabel(): ?string {
 			// Fallback
 			return $this->getEmail();
 		}
+		$personal = trim(explode('<', $personal)[0]); // Remove the email part if present
```

Comment:
I don't quite understand this line. Please elaborate: Why does the personal part contain `<`? Is this a bug in Horde?

Reply:
Some clients allow customising sender address such as ThunderBird, In that case Personal would be 
`custom label <custom.address@example.com>`  and from `$email` would still contain the original email.

Reply:
Ahh, now I get it. Thanks.

Code makes sense: https://3v4l.org/rAY8u